src/nixpacks/phase.rs

```diff
@@ -12,6 +12,7 @@ pub struct SetupPhase {
     pub archive: Option<String>,
     pub libraries: Option<Vec<String>>,
     pub apt_pkgs: Option<Vec<String>>,
+    pub cmd: Option<Vec<String>>,
```

Comment:
Rename the variable to be more descriptive. Also, if this is a vector of commands, I think the commands for other phases should be a vector as well to be consistent.

```suggestion
    pub cmds: Option<Vec<String>>,
```